cmd/pinniped-proxy/Dockerfile

```diff
@@ -12,12 +13,7 @@ ADD ./cmd/pinniped-proxy ./
 RUN rm ./target/release/deps/pinniped_proxy*
 RUN cargo build --release
 
-# Put it all together on a scratch image.
-# TODO: Turns out we'll need to include libssl on the scratch image, perhaps compiling libssl or copying dyn libs listed by ldd.
-# FROM scratch
-# COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
-# FROM bitnami/minideb:buster-snapshot-20201121T213529Z
-FROM debian:10.6-slim
+FROM bitnami/minideb:buster-snapshot-20210126T024633Z
 RUN apt-get update && apt-get install -y ca-certificates libssl1.1 && rm -rf /var/lib/apt/lists/*
```

Comment:
Not sure if you replied this before but why not using `scratch` and copy the `ca-certificates`? That's what we do with other services.

Reply:
It's not just the ca-certificates. The proxy dynamically links to libssl. There are ways to compile with static linking which I tried, but it becomes quite complicated (or uses a rando base image for the build), so I started simple with a minimal base image with libssl available.

Reply:
got it, it would be ideal to statically link it